packages/core/src/api/resolvers/entity/product-option-group-entity.resolver.ts

```diff
@@ -37,7 +37,10 @@ export class ProductOptionGroupEntityResolver {
         if (optionGroup.options) {
             options = optionGroup.options;
         } else {
-            const group = await this.productOptionGroupService.findOne(ctx, optionGroup.id);
+            const group = await this.productOptionGroupService.getOptionGroupForFieldResolver(
```

Comment:
this change is technically redundant because we already filter out the deleted ones on line 46.